### PR TITLE
make keywords bold in fix messages to slightly improve visibility

### PR DESCRIPTION
**Pull request by KazuCocoa**
How about making key-words bold in fix messages to improve visibility?
The name of tools/libraries users should install or PATH to set them up.

- after
image

- before
image

### CODE REVIEW

File: test/android-specs.js [hunk 1]
```diff
@@ -122,7 +122,7 @@ describe('android', function () {
       mocks.verify();
     });
     it('fix', async function () {
-      (await check.fix()).should.equal('bundletool.jar is used to handle Android App Bundle. Please read http://appium.io/docs/en/writing-running-appium/android/android-appbundle/ to install it');
+      (await check.fix()).should.equal(`${'bundletool.jar'.bold} is used to handle Android App Bundle. Please read http://appium.io/docs/en/writing-running-appium/android/android-appbundle/ to install it`);
```
Comment (mykola-mokhnach) [inline test/android-specs.js]: I'd rather check for simple text inclusion here to not overcomplicate tests with color modifiers

Comment (mykola-mokhnach) [reply test/android-specs.js]: same below

File: test/helper.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+/**
+ * Remove ANSI colors/styles
+ *
+ * @param {string} text
+ * @returns {string} The text which has no ANSI colors/styles
+ */
+function removeColors (text) {
+  return text.replace(/[\u001b\u009b][[()#;?]*(?:[0-9]{1,4}(?:;[0-9]{0,4})*)?[0-9A-ORZcf-nqry=><]/g, ''); // eslint-disable-line no-control-regex
```
Comment (mykola-mokhnach) [inline test/helper.js]: what a regex o_O

Comment (KazuCocoa) [reply test/helper.js]: yea... lol

**Pull request merged**